local-cli/generator-android/templates/package/MainActivity.java

```diff
@@ -24,7 +24,7 @@ protected String getMainComponentName() {
      */
     @Override
     protected boolean getUseDeveloperSupport() {
-        return BuildConfig.DEBUG;
+        return true;
```

Comment:
Good point.
BuildConfig is a gradle-ism, we need a cross-builder way to pass parameters like this.
Any ideas?